geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/pubsub/SubscriptionsIntegrationTest.java

```diff
@@ -39,9 +38,8 @@
   public static ExecutorServiceRule executor = new ExecutorServiceRule();
 
   @Test
-  @Ignore("GEODE-8515")
   public void pingWhileSubscribed() {
-    Jedis client = new Jedis("localhost", server.getPort());
+    Jedis client = new Jedis("localhost", server.getPort(), 1000000000);
```

Comment:
why a long timeout on this one but not on pingWithArgumentWhileSubscribed?

Reply:
Thank you for calling this out.  I had a long timeout for debugging and forgot to remove it.